ak_vendor/enums.py

```diff
@@ -13,6 +13,25 @@ class Meta:
         DEVKNOX = [1, 'Devknox']
 
 
+@choices
+class OWASPCATEGORIESEnum:
+    class Meta:
+        OWASP_CAT_A1 = [0, 'A1']
+        OWASP_CAT_A3 = [1, 'A3']
+        OWASP_CAT_A8 = [2, 'A8']
+        OWASP_CAT_M1 = [3, 'M1']
+        OWASP_CAT_M2 = [4, 'M2']
+        OWASP_CAT_M3 = [5, 'M3']
+        OWASP_CAT_M4 = [6, 'M4']
+        OWASP_CAT_M5 = [7, 'M5']
+        OWASP_CAT_M6 = [8, 'M6']
+        OWASP_CAT_M7 = [9, 'M7']
+        OWASP_CAT_M8 = [10, 'M8']
+        OWASP_CAT_M9 = [11, 'M9']
+        OWASP_CAT_M10 = [12, 'M10']
+        OWASP_CAT_UNKNOWN = [13, 'UNKNOWN']
```

Comment:
@subho007 @SubhamBhattacharjee You can remove `UNKNOWN`. The decorator adds unknown automagically `from orm_choices import choices_with_unknown as choices`

Reply:
OK Will do it. Thanks :+1: